joomla/joomla-cms
# Remove testing data from distribution

Pull Request for Issue #10330
#### Summary of Changes
- remove the testing files from the core stable versions
#### Testing Instructions
- run the build
- see that the testing files are not there
- run the installation
- see that there is no testing sample files option.

Thanks @brianteeman
